tests/test_h5store.py

```diff
@@ -757,12 +759,10 @@ def format_row(text, reducer):
         msg += format_row('median', numpy.median)
         msg += format_row('25 percentile', partial(numpy.percentile, q=25))
         msg += format_row('75 percentile', partial(numpy.percentile, q=75))
-        self.assertLess(
-            numpy.percentile(times, 25) / numpy.percentile(self.baseline_time, 75),
-            self.max_slowdown_vs_native_factor, msg)
+        assert numpy.percentile(times, 25) / numpy.percentile(self.baseline_time, 75) < self.max_slowdown_vs_native_factor, msg
```

Comment:
```suggestion
        assert numpy.percentile(times, 25) / numpy.percentile(self.baseline_time, 75) < \
            self.max_slowdown_vs_native_factor, msg
```
Lines should not be more than 100 characters long.